frontend/src/features/manageVehicles/components/form/TrailerForm.tsx

```diff
@@ -27,6 +28,7 @@ import {
   requiredMessage,
 } from "../../../../common/helpers/validationMessages";
 import { VEHICLES_ROUTES } from "../../../../routes/constants";
+import { Nullable } from "vitest";
```

Comment:
I think it's the wrong Nullable type. I'm guessing you're trying to use the one in the common types folder?

Reply:
as in import { Nullable } from "../../../../common/types/common";

Reply:
Updated! good catch!